src/main/java/com/google/devtools/common/options/Converters.java

```diff
@@ -457,6 +457,31 @@ public String getTypeDescription() {
     }
   }
 
+  /**
+   * A converter for variable assignments from the parameter list of a blaze command invocation.
+   * Assignments are expected to have the form "name=value", where names are defined to
+   * be as permissive as possible and the values should be floats.
+   */
+  public static class StringToFloatAssignmentConverter implements Converter<Map.Entry<String, Float>> {
+
+    @Override
+    public Map.Entry<String, Float> convert(String input) throws OptionsParsingException {
+      int pos = input.indexOf("=");
```

Comment:
Just use a static Splitter instead of hand-coding. Alternatively, make this a subclass of AssignmentConverter. Or even better, make AssignmentConverter a generic method with a converter lambda.